kernel/patches-4.4.x/0016-drivers-hv-Define-the-channel-type-for-Hyper-V-PCI-E.patch

```diff
@@ -40,10 +40,11 @@ diff --git a/include/linux/hyperv.h b/include/linux/hyperv.h
 index ae6a711dcd1d..10dda1e3b560 100644
 --- a/include/linux/hyperv.h
 +++ b/include/linux/hyperv.h
-@@ -1156,6 +1156,17 @@ int vmbus_allocate_mmio(struct resource **new, struct hv_device *device_obj,
+@@ -1155,6 +1155,17 @@ int vmbus_allocate_mmio(struct resource **new, struct hv_device *device_obj,
+ 			0xab, 0x99, 0xbd, 0x1f, 0x1c, 0x86, 0xb5, 0x01 \
```

Comment:
is this intended?  Looks odd.

Reply:
I noticed this too, but I used the same commands I always use. Maybe some subtle difference in how git handles whitespace in patch files...The kernel compiled fine, though.

Reply:
ok sounds good then 👍